### PR TITLE
 Add new recipe for CRAN package metaMA 

### PR DESCRIPTION
**Pull request by sblanck**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-metama/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+test:
+  commands:
+    - $R -e "library('metaMA')"      
+
+about:
+  home: https://CRAN.R-project.org/package=metaMA
+  license: GPL
+  summary: Combines either p-values or modified effect sizes from different studies to find differentially
+    expressed genes
+  license_family: GPL
```
Comment (bgruening) [inline recipes/r-metama/meta.yaml]: Can you please add:
`license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'  # [unix]`

File: recipes/r-metama/build.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+export DISABLE_AUTOBREW=1
```
Comment (bgruening) [inline recipes/r-metama/build.sh]: Interesting, that this is needed!

File: recipes/r-metama/meta.yaml (hunk 1 above)
Comment (sblanck) [reply recipes/r-metama/meta.yaml]: With this line, the Travis CI build failed :
https://travis-ci.org/bioconda/bioconda-recipes/jobs/313376197

**Comment by bgruening** [comment]
Thanks @sblanck! Sorry for the noise.

**Pull request merged**